etc/octavia_proxy.conf

```diff
@@ -27,7 +27,14 @@ log_name='validatetoken'
 [api_settings]
 bind_host = 0.0.0.0
 bind_port = 9876
-#region = eu-nl
+
+# This setting is for proper networking handling, it has default value for eu-de region.
+# Must be set for eu-nl or another regions.
+# region = eu-nl
+
+# This setting is mandatory for elbv3, it has default value for eu-de region.
+# Must be set for eu-nl or another regions.
+# base_az = eu-nl-01
```

Comment:
can we please name it default_az? It will match better the description as well. Also replace elbv3 with smth like dedicated load balancer in the comment